### ISSUE TITLE
A11y_.NETCore3.0(WPF)_WPF ControlsAndLayout_Controls Tooltip_ScreenReader: Screen reader is not announcing tooltip. 

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1462]
Copied from VSO: [951723](https://devdiv.visualstudio.com/DevDiv/_queries/edit/951723/)

Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:
Application Name: .NET Core 3.0 (WPF)

.NET Core SDK version:  3.0.100-preview8-013442

Windows Version: Windows10

ScreenReader Version: Narrator, NVDA 2019.1.1

Pre-requisite:

1) Download the latest .NET Core 3 daily build by going to this link.

2) Install Windows (x86 or x64) versions of sdk listed under Master (3.0.x Runtime) by clicking "Installer" link. 

Repro Steps:

1) Open Visual Studio.

2) Hit on File and select Open->Project/Solution.

3) Run exe file for "ControlsAndLayout" from the build app folder provided. 
4) Main Window for "ControlsAndLayout" solution will appear. 
5) From the Sample Library pane under Controls, select "Tooltip". 
6) Turn on Screen reader.
7) Navigate to Textbox using keyboard.

Actual:

When screen reader focus goes to Textbox, tooltip appears but it is not announced by Screen reader. 

Note: Similar Issue is observed with NVDA

Expected:

Tooltip should be announced by screen reader.

User Impact:

Screen reader users unable to understand about Tooltip information. 

Recommendations:

Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/Pages/Home.aspx

MAS Reference:

MAS 1.1.1

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

MAS1.3.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by rladuca** [comment]
Narrator does indeed read tooltips if you turn on:
![image](https://user-images.githubusercontent.com/20006987/64555243-bfe64000-d2f1-11e9-82e9-877c8dffe199.png)
